src/app/components/accordion/accordion.ts

```diff
@@ -281,11 +281,31 @@ export class AccordionTab implements AfterContentInit, OnDestroy {
 
         this.selectedChange.emit(this.selected);
         this.accordion.updateActiveIndex();
+        this.fixKeyboardNavigation();
         this.changeDetector.markForCheck();
 
         event.preventDefault();
     }
 
+    fixKeyboardNavigation() {
```

Comment:
I think you can optimize your code like this:

```
fixKeyboardNavigation() {
  const accordionTabs = this.accordion.el.nativeElement.querySelectorAll('.p-accordion-tab');
  accordionTabs.forEach(tab => {
    const content = tab.querySelector('.p-accordion-content');
    const focusableElements = DomHandler.getFocusableElements(content);

    focusableElements.forEach(element => {
      element.tabIndex = tab.classList.contains('p-accordion-tab-active') && !element.classList.contains('p-accordion-header-link') ? -2 : 0;
    });
  });
}
```
Check it first but I think should be work :)

Reply:
This works! Thanks, pushed changes to the PR